app/helpers/textual_mixins/textual_devices.rb

```diff
@@ -12,31 +12,37 @@ def textual_devices
     h
   end
 
+  def processor_description
+    _("%{total_cores} (%{num_sockets} Sockets x %{num_cores} Cores)") % {:total_cores => @record.cpu_total_cores,
+                                                                         :num_sockets => @record.num_cpu,
+                                                                         :num_cores => @record.cpu_cores_per_socket}
+  end
+
+  def cpu_attributes
```

Comment:
yea that sounds good to me, i'm going to apply in the next iteration of the refactoring